docs/examples.rst

```diff
@@ -263,11 +263,11 @@ CybOX type.
 
     <cyboxCommon:HashListType>
         <cyboxCommon:Hash>
-            <cyboxCommon:Type xsi:type="cyboxVocabs:HashNameVocab-1.0">MD5</cyboxCommon:Type>
+            <cyboxCommon:Type>MD5</cyboxCommon:Type>
```

Comment:
Why take these out? I didn't think `cyboxVocabs:HashNameVocab-1.0` was the default value. It makes for cleaner XML, but if I understand how vocabularies work, there is no relationship between this `MD5` value and the value in the vocab. Am I missing something?